### DISCUSSION TITLE
Cant get neolink to run via command prompt

### DISCUSSION THREAD

**Discussion by tessanadik** [thirtythreeforty/neolink#289] [Q&A]
I have windows 1o pro running on my intel nuc, which is also where I run blue iris (Has is on a raspi4). I downloaded and installed gstreamer 1.0 msvc x86_64. I then added this path "%GSTREAMER_1_0_ROOT_X86_64%\bin" but the entire path from C:\ on into the environment variables under "path". I then created a txt file named "my_config.toml" with the following entry saved in the same folder as where I unzipped neolink:

bind: = "0.0.0.0" 
 [[cameras]]
 name = "cameraname"
 username = "admin"
 password = "password"
 address = "myip
 stream = "both"

 I then launch command prompt and enter "neolink rtsp --config my_config.toml" that   'neolink' is not recognized as an internal or external command

**Comment by QuantumEntangledAndy** [comment]
You need to use the cd command to change the directory inside cmd to the directory that contains neolink.

**Reply by tessanadik** [reply]
I figured that part out. Now I have this issue where it says "goobject-2.0.0.... was not found" Could it be how I added gstream to the path? https://imgur.com/qFFxmP6

**Reply by QuantumEntangledAndy** [reply]
Looks like gstreamer is not setup correctly. Did you add the actuall GSTREAMER_1_0_ROOT_X86_64 variable. Not just the one in the path that you have shown.

Also this is the command I meant by using cd

```cmd
cd C:\Users\tessa\OneDrive\Desktop\release-windows-2019\
neolink rtsp --config neolink.toml
```

You may have better luck installing gstreamer via chocolatey as it handles setting up the environmental variables.

**Comment by TheNerdSquad** [comment]
I had issues, then installed the GStreamer via Choco and it added all the correct entries
Then ensure to do a restart, otherwise it wont see the updated link
Also ensure to run command prompt or powershell inside of the folder which contains your .toml or .yaml
